Misc/NEWS.d/next/Library/2017-09-19-13-29-29.bpo-26510.oncW6V.rst

```diff
@@ -0,0 +1,3 @@
+Fix regression in ``add_subparsers``: subparsers are now required by
+default. For optional subparsers, ``required=False`` may be passed as a
+keyword argument to ``add_subparsers``.  Patch by Anthony Sottile
```

Comment:
@Mariatta Could you tell me if it’s the current practice to include contributors’ names in NEWS entries?

@asottile Add yourself to ACKS if you’re not already there.

Reply:
@merwok Yes to "Patch by <contributor>" in news entries.

And also +1 to adding @asottile Misc/ACKS.

Reply:
`Patch by <contributor>` should end with a period. :)

Reply:
I attempted to follow [this example](https://docs.python.org/devguide/committing.html#what-s-new-and-news-entries) --  if not perhaps that should be updated.

[Found myself!](https://github.com/python/cpython/blob/525269430a3f9fbb7287e4bb6b365ac216004980/Misc/ACKS#L1489)

Reply:
Thanks for the help Mariatta and for the link Anthony!